src/Data/Distributive/Generic.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE FlexibleContexts #-}
```

Comment:
MAke sure to mark this as `Trustworthy`, since it imports `Data.Coerce`, which will be inferred `Unsafe`.